Android/quickstart-kotlin/app/src/main/assets/immersiveReader.html

```diff
@@ -0,0 +1,34 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. -->
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <script type="text/javascript" src="https://contentstorage.onenote.office.net/onenoteltir/immersivereadersdk/immersive-reader-sdk.0.0.3.js"></script>
+</head>
+<body>
+    <script type="text/javascript">
+        function handleLaunchImmersiveReader(message) {
+            if (!message) {
+                Android.showToast('Message is null or undefined!');
+            } else {
+
+                // A simple declarative function used to close the Immersive Reader WebView via @JavaScriptInterface
+                const exitCallback = function() {
+                    Android.immersiveReaderExit();
+                }
+
+                // A method cannot be passed into the message JSON, so we assign it here as there's
+                // no need to post a message to the parent of the Immersive Reader iframe because

```

Comment:
what does post messaging have to do with the on exit callback?

Reply:
The exit handler is called I JS, then @JavaScriptInterface calls it in code to close the WebView.

Reply:
Changed the comment to avoid any potential confusion.